src/utils/Scale.js

```diff
@@ -147,10 +131,21 @@ export function invertPointScale(scale, rangeValue) {
   // shim until d3.scalePoint.invert() is implemented for real
   // given a value from the output range, returns the *nearest* corresponding value in the input domain
   const rangePoints = domain.map(domainValue => scale(domainValue));
-  const nearestLeftPointIndex = indexOfClosestLeftNumberInList(
-    rangeValue,
-    rangePoints
-  );
 
-  return domain[nearestLeftPointIndex];
+  if (rangePoints.length <= 1) {
+    return domain[0];
+  }
+
+  const isDescending = rangePoints[0] > rangePoints[1];
+
+  // _.sortedIndex works on ascending lists only
+  // so reverse if working with descending list
+  if (isDescending) {
+    domain.reverse();
+    rangePoints.reverse();
+  }
```

Comment:
FWIW I don't think you need this descending/reverse logic anymore since you're not assuming anything about sort order.

Reply:
that's a good point, will actually opt to update the code above and just use lodash's sortedIndex